src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -677,17 +683,17 @@
 
 
 #infoBubble {
-  background-color: #e1f3f8;
+  background-color: $color-aqua-lightest;
```

Comment:
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L41